src/libvmaf/Makefile

```diff
@@ -1,8 +1,10 @@
 LIBVMAF = libvmaf.a
+LIBVMAF_DYN = libvmafdyn.so
```

Comment:
could we just call it `libvmaf.so`?

Reply:
Yeah, we can rename if we want to. Just gave this name for simplicity to avoid confusion with libvmaf.a when doing -lvmaf.

Reply:
The suffix for these is platform dependent. `.dylib` on MacOS or `.dll` on Windows for example.

See line 3b here: http://web.archive.org/web/20110720003722/http://www.fortran-2000.com/ArnaudRecipes/sharedlib.html

It is better to use a build system like `CMake` or `meson` to handle this. I will be sending a PR soon to address this.

Reply:
sure thing, that will help make building cleaner!